integration-tests/environments/react-native/src/App.js

```diff
@@ -24,6 +24,9 @@
  * @flow
  */
 
+// Polyfill needed for for `fetch` and `ReadableStream`.
```

Comment:
Great, we can use the syntax you have in `test-migration-master` so that we only import exactly what we need.

For the description in #5410, we could add that this variant also is a possibility:
```js
import { polyfill } from "react-native-polyfill-globals";
polyfill();
```

But I do agree that simplifying this for users would be better!